src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/CborHelpers.netstandard.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Formats.Cbor
+{
+    internal static partial class CborHelpers
+    {
+        private const long UnixEpochTicks = 719162L /*Number of days from 1/1/0001 to 12/31/1969*/ * 10000 * 1000 * 60 * 60 * 24; /* Ticks per day.*/
+
+        public static readonly DateTimeOffset UnixEpoch = new DateTimeOffset(UnixEpochTicks, TimeSpan.Zero);
+
+        public static unsafe int GetBytes(Encoding encoding, ReadOnlySpan<char> source, Span<byte> destination)
+        {
+            if (source.IsEmpty || destination.IsEmpty)
+            {
+                return 0;
+            }
+
+            fixed (char* charPtr = source)
+            fixed (byte* bytePtr = destination)
+            {
+                return encoding.GetBytes(charPtr, source.Length, bytePtr, destination.Length);
+            }
+        }
+
+        public static unsafe int GetByteCount(Encoding encoding, ReadOnlySpan<char> chars)
+        {
+            if (chars.IsEmpty)
+            {
+                return 0;
+            }
+
+            fixed (char* charPtr = chars)
+            {
+                return encoding.GetByteCount(charPtr, chars.Length);
+            }
+        }
+
+        public static unsafe int GetChars(Encoding encoding, ReadOnlySpan<byte> source, Span<char> destination)
```

Comment:
I wonder if some of these have already be implemented as part of other ns2.0 ports. If so, would it make sense to extract them to a common source file, so that they can get test coverage from multiple angles?

Reply:
I searched for `GetChars` methods that take a `ROS<char>` in any parameter but that didn't return any netstandard-specific files.
This is the query I used in vscode (with regex on): 
```
GetChars\([a-zA-Z0-9,<> ]*ReadOnlySpan<byte>
files to include: ./src/**/*.cs`
```
The closest thing I found was this GetBytes definition but it doesn't look ns-specific: https://cs.github.com/dotnet/runtime/blob/d0e2dac3a828b6afda5cff812b7b09b9dba213b7/src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnCharacterStringEncodings.cs?q=def%3AGetChars+language%3AC%23#L53